client/connect-account-page/index.js

```diff
@@ -11,6 +11,7 @@ import { render, useState, useEffect } from '@wordpress/element';
  */
 import OnboardingLocationCheckModal from './modal';
 import Page from 'components/page';
+import ErrorBoundary from 'components/error-boundary';
```

Comment:
I'm wrapping the content of all pages within ErrorBoundary. We still have to define how we'd like to track errors, but it might be better to wrap the page's contents directly in the [Page component](https://github.com/Automattic/woocommerce-payments/blob/develop/client/components/page/index.tsx#L43) instead of doing it on each page.

I'm doing it this way for now in case each page would trigger different actions. But if this won't be the case, I think having it within the Page component instead would be neater. Glad to learn if some other approach would be better, though!